containers/local.env

```diff
@@ -1,5 +1,5 @@
 INDEX_DB_LOCATION=/tmp/lbdev
-FILES_PATH=/tmp/lbdev/files
+FILES_PATH=/tmp/lbdev/docs
```

Comment:
becomes consistent with `prod.env` so that a tar'd backup of prod can be tested locally